pkg/k8s/const.go

```diff
@@ -30,4 +30,9 @@ const (
 	// EnvNodeNameSpec is the environment label used by Kubernetes to
 	// specify the node's name.
 	EnvNodeNameSpec = "K8S_NODE_NAME"
+
+	// compatibleK8sVersions is the range of k8s versions this cilium is able to
+	// work with. It will change as we add new support or deprecate older k8s
+	// versions.
+	compatibleK8sVersions = "> 1.6, < 1.11"
```

Comment:
< 1.12

Reply:
are we tested to work with 1.11? I wasn't sure how strict to make this. Would it be a "only works with things we tested" or "this will probably work". If the latter, we could just have "> 1.6" since it's simpler.

Reply:
Chatted offline. I removed the upper bound to avoid forcibly breaking in the future.